tests/Behat/Context/Ui/RefundingContext.php

```diff
@@ -31,7 +32,10 @@ public function __construct(
      */
     public function wantToRefundSomeUnitsOfOrder(string $orderNumber): void
     {
-        $this->orderRefundsPage->open(['orderNumber' => $orderNumber]);
+        try {
+            $this->orderRefundsPage->open(['orderNumber' => $orderNumber]);
+        } catch (UnexpectedPageException $exception) {
```

Comment:
Do we really need to catch it if we do nothing about that? This way some exception would be just thrown in one of the next steps